src/rohan/SwitchBanking.java

```diff
@@ -0,0 +1,159 @@
+package rohan;
+
+import java.util.Scanner;
+
+/*A person has total 4 accounts in ICICI, HDFC, BOB, SBI one in each.
+          a) Person can debit or credit amount from respective bank.
+              Hint : Switch case is expected here. [Please watch today's recording again, in case of confusion.]
+
+          b) A Method to show total balance [ICICI balance + HDFC balance + BOB balance + SBI Balance]
+
+          c) A Method to show total credit, debit operations 
+                 output : Total credit operation - 7 times
+                              Total debit operation - 5 times
+
+         d) A Method to show individual bank credit operations.
+                output : ICICI bank Credit Operation - 2
+                         HDFC bank Credit Operation - 2
+		                 SBI bank Credit Operation - 1
+                         BOB bank Credit Operation - 4
+
+         e) A Method to show individual bank debit operations.
+                output : ICICI bank Debit Operation - 2
+                         HDFC bank Debit Operation - 2
+		                 SBI */
+
+public class SwitchBanking {
+	double iciciTotalBalance, hdfcTotalBalance, bobTotalBalance,sbiTotalBalance;
+	int iCreditCnt,hCreditCnt,bCreditCnt,sCreditCnt;
+	int iDebitCnt,hDebitCnt,bDebitCnt,sDebitCnt;
+	
+	void setTotalBalance(double iTotalAmt, double hTotalAmt, double bTotalAmt, double sTotalAmt) {
+		iciciTotalBalance = iTotalAmt;
+		hdfcTotalBalance = hTotalAmt;
+		bobTotalBalance = bTotalAmt;
+		sbiTotalBalance = sTotalAmt;
+	}
+	
+	void printMessage(String bName, String operation) {
+		System.out.println("----Transcation Terminated----");
+		System.out.println("Your "+bName+" account balance is: "+iciciTotalBalance+" Rupees. Kindly enter correct amount to be "+operation+"ed");
+	}
+	
+	void performTransactions(String bankName,String operation, double amount) {
+		switch(operation) {
+		case "Credit": 
+			switch(bankName) {
+			case "ICICI":
+						iciciTotalBalance += amount;
+						iCreditCnt++;
+						break;
+			case "HDFC":
+						hdfcTotalBalance += amount;
+						hCreditCnt++;
+						break;
+			case "BOB":
+						bobTotalBalance += amount;
+						bCreditCnt++;
+						break;
+			case "SBI":
+						sbiTotalBalance += amount;
+						sCreditCnt++;
+						break;
+			default: System.out.println("Invalid bank Name specified");	
+			}
+			break;
+		case "Debit":
+			switch(bankName) {
+			case "ICICI":
+						if(amount<iciciTotalBalance) {
+							iciciTotalBalance -= amount;
+							iDebitCnt++;
+						}
+						else
+							printMessage("ICICI", "Debit");
+						break;
```

Comment:
Here, You can pass bankName, no need to pass "ICICI" as value, bcoz think when come to this case when case machted ICICI right. so in bankName variable we already have that value as ICICI/BOB etc. 
just correct thinking here